vendor/modules.txt

```diff
@@ -0,0 +1,59 @@
+# github.com/golang/protobuf v1.3.2
```

Comment:
nit: not sure if adding all vendored dependencies was indented here since `master` ([14aff55 at time of comment](https://github.com/dapr/go-sdk/tree/14aff5543faa60837ab02a1eb97cdc8a14a33455)) has a `go.mod` and no `vendor` directory.  Might suggest separating the vendoring of deps from a feature proposal.  Just an opinion, feel free to ignore.

*Note: Just a community member, not a Microsoft employee or dapr maintainer*

Reply:
There are strong opinions on both sides of the issue re vendor dir in git, in this case, the project is voletile right now (ramping up to 0.7 release) so I vendored deps with this PR to minimize test time issues.